apps/examples/e2e/shared-e2e.ts

```diff
@@ -0,0 +1,46 @@
+import { PlaywrightTestArgs, PlaywrightWorkerArgs } from '@playwright/test'
+import { App } from '@tldraw/tldraw'
+
+export function sleep(ms: number) {
+	return new Promise((resolve) => setTimeout(resolve, ms))
+}
+
+// export async function expectPathToBe(page: Page, path: string) {
+// 	expect(await page.evaluate(() => app.root.path.value)).toBe(path)
+// }
+
+// export async function expectToHaveNShapes(page: Page, numberOfShapes: number) {
+// 	expect(await page.evaluate(() => app.shapesArray.length)).toBe(numberOfShapes)
+// }
+
+// export async function expectToHaveNSelectedShapes(page: Page, numberOfSelectedShapes: number) {
+// 	expect(await page.evaluate(() => app.selectedIds.length)).toBe(numberOfSelectedShapes)
+// }
+
+declare const app: App
+
+export async function setup({ page, context }: PlaywrightTestArgs & PlaywrightWorkerArgs) {
+	await page.goto('http://localhost:5420/')
+	await page.waitForSelector('.tl-canvas')
+	await page.evaluate(() => (app.enableAnimations = false))
+	await context.grantPermissions(['clipboard-read', 'clipboard-write'])
```

Comment:
I guess we could do this conditionally so we can also test cases where users don't give the permissions (or just disable permissions in those tests).

Reply:
Ah, I'm going to trim that out actually. The only place where they were being used was clipboard, but that still doesn't seem to work in playwright.